src/x86_64.rs

```diff
@@ -0,0 +1,331 @@
+// Copyright 2018-2019 CrowdStrike, Inc.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Portions Copyright 2018 Cloudbase Solutions Srl
+// SPDX-License-Identifier: Apache-2.0
+//
+// Portions Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the THIRD-PARTY file.
+
+use std::mem::size_of;
+
+///
+/// Use kvm_bindings behind the scenes as these are architectural structures and
+/// not actually KVM-dependent, but export as generically-named data
+/// structures to be consumed by any VMM's vCPU implementation
+///
+
+/// Single MSR to be read/written
+///
+/// pub struct kvm_msr_entry {
```

Comment:
Any reason to have the kvm structures definitions as comments? The default rust documentation will show the inner structure. To check how it looks you can run ```cargo doc --open```.

Reply:
Also related to an older/outdated commit, comments were eliminated with the commit that removed the kvm_bindings dependency from Windows builds. Nevertheless, appreciate the general guideline and info about `cargo doc` for next time.